src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FC, useState, useCallback, useMemo } from 'react';
+import debounce from 'lodash.debounce';
+import cn from 'classnames';
+
+import { peopleFromServer } from '../../data/people';
+import { Person } from '../../types/Person';
+import { filterPeople } from '../../utils/filterPeople';
+import { DropdownMenu } from '../DropdownMenu/DropdownMenu';
+
+type Props = {
+  selectedPerson: Person | null;
+  onSelect: React.Dispatch<React.SetStateAction<Person | null>>;
```

Comment:
I think - this will be better and more readable, right?
```suggestion
  onSelect: (person: Person | null) => void;
```